thinkhazard/static/less/report.less

```diff
@@ -229,3 +229,25 @@ body {
     margin-bottom: 4px;
   }
 }
+
+.style-pdf-overview-btn {
+  padding: 7px 10px;
+  margin-top: 15px;
+  border: 1px solid #d9d9d9;
+  font-size: 14px;
+  .border-top-radius(5px);
+  .border-right-radius(5px);
+  .border-left-radius(5px);
+  .border-bottom-radius(5px);
```

Comment:
I don't think those mixins are useful for what you want to do.
The following is enough in my opinion:
```css
border-radius: 5px;
```